src/app/api/quality_matrix/models.py

```diff
@@ -1,5 +1,9 @@
+from __future__ import annotations
```

Comment:
not needed here?

Reply:
Ayayayay, removed as well as other unnecessary occurences